proposals/extensions.md

```diff
@@ -535,7 +749,7 @@ For purposes of member lookup, a type `T` is considered to have the following b
 - If `T` is an *interface_type*, the base types of `T` are the base interfaces of `T` and the class type `object`.
 - If `T` is an *array_type*, the base types of `T` are the class types `System.Array` and `object`.
 - If `T` is a *delegate_type*, the base types of `T` are the class types `System.Delegate` and `object`.
-- \***If `T` is an *extension_type*, the base types of `T` are the base extensions of `T`.**
+- \***If `T` is an *extension_type*, the base types of `T` are the base extensions of `T` and the extended type of `T`.**
```

Comment:
What about the extended types of the base extensions - they can be different, right?

Also PR https://github.com/dotnet/roslyn/pull/71779 has "and its base types." in the spec quote which isn't here.